include/arbor/util/uninitialized.hpp

```diff
@@ -43,15 +43,15 @@ class uninitialized {
     using const_rvalue_reference= const X&&;
 
     pointer ptr() {
-        // COMPAT: xlC 13.1.4 workaround:
+        // COMPAT: xlC 13.1.4 workaround (still broken in 14.1.0):
```

Comment:
Are we just going to remove this and refuse to compile with xlc?
xlc is worse than useless: generates incorrect code that is 10x slower than gcc.

Reply:
We will! I wanted to make the xlC deprecation its own PR. (It could be the next one, even.)